client/post-editor/editor-drawer-well/style.scss

```diff
@@ -7,7 +9,6 @@
 }
 
 .editor-drawer-well__placeholder {
-	@extend .card;
```

Comment:
`@extend` is causing some problems when used in combination with Webpack because it shifts code around and confuses styles applied somewhere else.

Interestingly it seems like this `@extend` call doesn't apply in the current production CSS though. You can check if you create a new post and look at the CSS for the featured image placeholder. No card styles are applied there at all. I'm not exactly sure why though and this probably needs further investigation.